search/includes/classes/class-search.php

```diff
@@ -916,6 +879,10 @@ public function filter__ep_do_intercept_request( $request, $query, $args, $failu
 			// Return a generic VIP Search WP_Error instead of the one from wp_remote_request
 			return new \WP_Error( 'vip-search-upstream-request-failed', 'There was an error connecting to the upstream search server' );
 		} else {
```

Comment:
We can probably drop this `else` statement, as the `if` branch ends with `return`.

Reply:
No longer relevant.